website/docs/r/schema_template_service_graph.html.markdown

```diff
@@ -33,16 +30,18 @@ resource "mso_schema_template_service_graph" "test_sg" {
 * `schema_id` - (Required) Schema ID where Service Graph to be created.
 * `template_name` - (Required) Template Name where Service Graph to be created.
 * `service_graph_name` - (Required) Name of Service Graph.
-* `service_node_type` - (Required) Type of Service Node attached to this Graph. Allowed values are `firewall`, `load-balancer` , `other`.
+* `service_node_type` - (Optional) **Deprecated**. Type of Service Node attached to this Graph. Allowed values are `firewall`, `load-balancer` , `other`.
 * `description` - (Optional) Description of Service Graph.
-* `site_nodes` - (Optional) List of maps to provide Site level Node association. This maps should be provided if site is associated with template.
-* `site_nodes.site_id` - (Optional) Site-Id Attached with the template. Where Service Graph will be created. This parameter is required when site is attached with the Template.
-* `site_nodes.tenant_name` - (Optional) Name of Tenant holding the Service Node. This parameter is required when site is attached with the Template.
-* `site_nodes.node_name` - (Optional) Name of Site level Service Node/Device Name. This parameter is required when site is attached with the Template.
+* `service_node` - (Required) List of service nodes attached to Service Graph.
+    * `service_node.type` - (Required) Type of Service Node attached to the Service Graph. Allowed values are `firewall`, `load-balancer` , `other`.
+
+
+## NOTE ##
+The `site_nodes` parameters are removed from Template level Service Graph resource.
 
 ## Attribute Reference ##
 
-The only Attribute exposed for this resource is `id`. Which is set to the Name of Service Graph created.
+The only Attribute exposed for this resource is `id`. Which is set to the name of Service Graph created.
```

Comment:
Is this correct? because we set the id to d.SetId(fmt.Sprintf("%s/templates/%s/serviceGraphs/%s", schemaId, templateName, graphName))

Reply:
changed to id